go/v1beta1/storage/queries.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2017 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+const (
+	createTables = `
```

Comment:
How is the long term upgrade story going to be handled here. Is the plan just for people to drop the tables and recreate them?

Reply:
Yes, for now. Long-term, this PostgreSQL implementation of the Grafeas API will not be maintained by the core Grafeas team, but by other contributors, and it will be moved to `github.com/grafeas/grafeas-pgsql`.